tests/integration/targets/azure_rm_loganalyticsworkspace/tasks/main.yml

```
@@ -159,9 +159,207 @@
   azure_rm_loganalyticsworkspace:
     name: "{{ name_rpfx }}"
     resource_group: "{{ resource_group }}"
-    retention_in_days: "{{ retention_days }}"
+    retention_in_days: 90
```

Why change this value?
```suggestion
    retention_in_days: "{{ retention_days }}"   ---- 60
```

---

Lines 363 to 366 should be inserted between lines 162 and 163

---

yeah that was an oversight because my testing does not have that variable :D

---

Line 9 has define this variables! Please restore this!